.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/VectorStores/AzureAddFileToVectorStoreOperation.Protocol.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ClientModel.Primitives;
+
+namespace Azure.AI.OpenAI.VectorStores;
+public partial class AzureAddFileToVectorStoreOperation : AddFileToVectorStoreOperation

```

Comment:
Same question about whether this needs to be public?  If there's a good why, that's cool, I just don't have that in my head currently.

Reply:
made internal